_episodes/11-connecting.md

```diff
@@ -0,0 +1,427 @@
+
+## Looking Around Your Remote Home
+
+Very often, many users are tempted to think of a high-performance computing
+installation as one giant, magical machine. Sometimes, people will assume that
+the computer they've logged onto is the entire computing cluster. So what's
+really happening? What computer have we logged on to? The name of the current
+computer we are logged onto can be checked with the `hostname` command. (You
+may also notice that the current hostname is also part of our prompt!)
+
+```
+{{ site.remote.prompt }} hostname
+```
+{: .language-bash}
+
+```
+{{ site.remote.host }}
+```
+{: .output}
+
+So, we're definitely on the remote machine. Next, let's find out where we are
+by running `pwd` to **p**rint the **w**orking **d**irectory.
+
+```
+{{ site.remote.prompt }} pwd
+```
+{: .language-bash}
+
+```
+{{ site.remote.homedir }}/{{ site.remote.user }}
+```
+{: .output}
+
+Great, we know where we are! Let's see what's in our current directory:
+
+```
+{{ site.remote.prompt }} ls
+```
+{: .language-bash}
+
+The system administrators may have configured your home directory with some
+helpful files, folders, and links (shortcuts) to space reserved for you on
+other filesystems. If they did not, your home directory may appear empty. To
+double-check, include hidden files in your directory listing:
+
+```
+{{ site.remote.prompt }} ls -a
+```
+{: .language-bash}
+```
+  .            .bashrc           id_ed25519.pub
+  ..           .ssh
+```
+{: .output}
+
+In the first column, `.` is a reference to the current directory and `..` a
+reference to its parent (`{{ site.remote.homedir }}`). You may or may not see
+the other files, or files like them: `.bashrc` is a shell configuration file,
+which you can edit with your preferences; and `.ssh` is a directory storing SSH
+keys and a record of authorized connections.
+
+### Install Your SSH Key
+
+> ## There May Be a Better Way
+>
+> Policies and practices for handling SSH keys vary between HPC
+> clusters: follow any guidance provided by the cluster
+> administrators or documentation. In particular, if there is an
+> online portal for managing SSH keys, use that instead of the
+> directions outlined here.
+{: .callout}
+
+If you transferred your SSH public key with `scp`, you should see
+`id_ed25519.pub` in your home directory. To "install" this key, it must be
+listed in a file named `authorized_keys` under the `.ssh` folder.
+
+If the `.ssh` folder was not listed above, then it does not yet
+exist: create it.
+
+```
+{{ site.remote.prompt }} mkdir ~/.ssh
+```
+{: .language-bash}
+
+Now, use `cat` to print your public key, but redirect the output, appending it
+to the `authorized_keys` file:
+
+```
+{{ site.remote.prompt }} cat ~/id_ed25519.pub >> ~/.ssh/authorized_keys
+```
+{: .language-bash}
+
+That's all! Disconnect, then try to log back into the remote: if your key and
+agent have been configured correctly, you should not be prompted for a password.
```

Comment:
Right, the intent here is that the SSH Agent always prompts for the _key_ password the first time it's loaded, but not thereafter. We should clarify, but retain this, because re-prompts indicate an Agent failure.